### ISSUE TITLE
how

### ISSUE THREAD

**Issue by memetree0** [Swag1400/java-discord-token-logger#1]
can u help idk how to open it how it works if u can help me plz contact me

**Comment by BruzzTheHuzz0** [comment]
https://github.com/BruzzTheHuzz0/Discord-Token-Logger revamped version :)

**Comment by Skidddd22** [comment]
omg memetree0 the epic 0b0t hacker pls give screenie for fame and big name

**Comment by Skidddd22** [comment]
bruzzthehuzz omg another big namer pls screenie plssssssssss

**Comment by K3maaa** [comment]
@quickdaffy  its a 4044444444444444444

**Comment by scary-harry-69** [comment]
> @quickdaffy its a 4044444444444444444

eww kema you fucking faggot trying to get a token logger to work so you can token log someone rofl

**Comment by memetree0** [comment]
> omg memetree0 the epic 0b0t hacker pls give screenie for fame and big name

yes yes XDD nah i need to use for other shit not iligal skiddi shit

**Comment by tman10001** [comment]
in what way could you use this in a legal way

**Comment by AstroOrbis** [comment]
Pranking friends

**Comment by FunWithAlbiYT** [comment]
yes
